pages/api/exercises.ts

```diff
@@ -0,0 +1,37 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import query from "../../db";
+import NextCors from "nextjs-cors";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  await NextCors(req, res, {
+    // Options
+    methods: ["GET", "HEAD", "PUT", "PATCH", "POST", "DELETE"],
+    origin: "*",
```

Comment:
when the front end is deployed, do we need to provide the front end link as the origin for extra security?

Reply:
Frontend has not been deployed yet.